Platform.PCL/Realm.Sync.PCL/UserPCL.cs

```diff
@@ -28,6 +28,46 @@ namespace Realms.Sync
     /// </summary>
     public class User : IEquatable<User>
     {
+        #region static
+
+        /// <summary>
+        /// Gets the currently logged-in user. If none exists, null is returned. If more than one user is currently logged in, an exception is thrown.
```

Comment:
Definition from the Java docs should be included.

 _ The current user is the last user who logged in and credentials are not expired yet_

It disambiguates the issue of what makes a given user _Current_ if you consider multiple servers being used.